contrib/submit-nfs-job/src/App/TensorflowDistributedJob.tsx

```diff
@@ -154,6 +164,8 @@ echo "All workers are: $PAI_TASK_ROLE_worker_HOST_LIST"
   useEffect(() => {
     onChange(new TensorflowDistributedJob(
       name,
+      image,
+      vc,
```

Comment:
please also add these 2 field as dependencies of `useEffect`

Reply:
thanks, I updated the code